### ISSUE TITLE
Shortcut keys overwritten every time in Intellij IDEA 2024.1.1

### ISSUE THREAD

**Issue by shme7ev** [continuedev/continue#1235]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:Ubuntu 22.04
- Continue: 0.46
- IDE: Intellij IDEA Community 2024.1.1
```

### Description

Shortcut keys overwritten with Continue mappings every time on restart in Intellij IDEA 2024.1.1 - for example, Ctl-Shift-J for joining lines. 

### To reproduce

1. Reassign Ctl-Shift-J back to Join lines.
2. Restart IDEA.
3. Ctl-Shift-J is again assigned to Continue action instead of Join lines

**Comment by sestinj** [comment]
Thanks for bringing this up. I just made a change [here](https://github.com/continuedev/continue/commit/d117d7fc191e601ace4199ed69b6d9a7a10007a8) so that once you remove the Continue shortcut once, it won't try to override again. Will be available in the next release!

**Comment by sestinj** [comment]
This change is now available in the latest EAP release, 0.0.47
